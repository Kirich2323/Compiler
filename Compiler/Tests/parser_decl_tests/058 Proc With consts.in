procedure name();
const
    a = 10;
    b : float = 0.5;
begin
end;