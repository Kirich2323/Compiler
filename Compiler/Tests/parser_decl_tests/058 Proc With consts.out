main
----------
char                     type 
float                    type 
integer                  type 
name                     proc 
 arguments:
 locals:
    a               const integer              10
    b                 const float             0.5
